### PR TITLE
Microbenchmarks for RSocket gateway

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**

### CODE REVIEW

File: Benchmarks/pom.xml [hunk 1]
```diff
@@ -12,13 +13,27 @@
...
+      <artifactId>scalecube-gateway-websocket</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.scalecube</groupId>
+      <artifactId>scalecube-gateway-examples</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.scalecube</groupId>
```
Comment (segabriel) [inline Benchmarks/pom.xml]: remove it

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/websocket/RequestOneBenchmarksRunner.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package io.scalecube.gateway.benchmarks.websocket;
+
+public class RequestOneBenchmarksRunner {
```
Comment (segabriel) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/websocket/RequestOneBenchmarksRunner.java]: remove it

File: Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/websocket/WebsocketState.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import io.scalecube.gateway.websocket.WebsocketServer;
+import io.scalecube.services.Microservices;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetSocketAddress;
+import java.time.Duration;
+
+public class WebsocketState extends BenchmarksState<WebsocketState> {
```
Comment (segabriel) [inline Benchmarks/src/main/java/io/scalecube/gateway/benchmarks/websocket/WebsocketState.java]: remove it

File: Core/pom.xml [hunk 4]
```diff
@@ -27,6 +27,10 @@
       <groupId>io.scalecube</groupId>
       <artifactId>scalecube-services-api</artifactId>
     </dependency>
+    <dependency>
```
Comment (segabriel) [inline Core/pom.xml]: remove it

**Comment by segabriel** [comment]
Add requestMany case

**Comment by segabriel** [comment]
created new one https://github.com/scalecube/scalecube-gateway/pull/56

**Comment by artem-v** [comment]
Closing in favour or https://github.com/scalecube/scalecube-gateway/pull/86/

**Pull request closed without merging**